core/src/main/scala/kafka/tools/StorageTool.scala

```diff
@@ -266,19 +300,19 @@ object StorageTool extends Logging {
   }
 
   private def getUserScramCredentialRecord(
-    mechanism: String,
-    config: String
-  ) : UserScramCredentialRecord = {
+                                            mechanism: String,
+                                            config: String
+                                          ) : UserScramCredentialRecord = {
```

Comment:
Please preserve the old indentation. In Kafka that is how we indent parameters to a method that span multiple lines.